app/Nova/Resources/UdbOrganizer.php

```diff
@@ -13,11 +12,11 @@
 use Laravel\Nova\Http\Requests\NovaRequest;
 
 /**
- * @mixin OrganizerModel
+ * @mixin UdbOrganizer
  */
-final class Organizer extends Resource
+final class UdbOrganizer extends Resource
 {
-    public static string $model = OrganizerModel::class;
+    public static string $model = UdbOrganizer::class;
```

Comment:
I think this breaks the Nova view, it should be `UdbOrganizerModel`, this also needs changing on line 15